Programming_Problems 4

```diff
@@ -1,5 +1,7 @@
-let n = prompt("Number n = ?");
-console.log(n);
-let x = 1  + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9;
-console.log(x);
-console.log(x + + n);
+var num = prompt("Number num = ?");
+var sum = 0;
+for(var i = 1; i <= num; i++) {
+    sum += i; }
```

Comment:
Minor formatting comment: Put the closing brace } of the loop on a new line.

Reply:
okay Sir.
Thanks a lot